lib/in_memory.ml

```diff
@@ -100,7 +100,8 @@ module Memory = struct
 
   let unshare share =
     List.iter (fun grant -> remove individual_pages grant) share.grants;
-    remove big_mapping (List.hd share.grants)
+    remove big_mapping (List.hd share.grants);
+    Lwt.return ()
```

Comment:
Extremely nitpick-y, but could we get an Lwt.return_unit here?

Reply:
Done (BTW, does clicking Resolve conversation on GitHub send a notification?).